src/main/java/jenkins/security/plugins/ldap/FromUserRecordLDAPGroupMembershipStrategy.java

```diff
@@ -51,6 +55,7 @@
 public class FromUserRecordLDAPGroupMembershipStrategy extends LDAPGroupMembershipStrategy {
 
     private static final Logger LOGGER = Logger.getLogger(FromUserRecordLDAPGroupMembershipStrategy.class.getName());
+    private static final String USER_SEARCH_FILTER = "({0}={1})";
```

Comment:
Parameters are substituted so this becomes `(getAttributeName()=groupDN)`. We could restrict it by adding and `&` with `(objectclass=inetOrgPerson)`, but I'm not sure if that would work with ActiveDirectory. Should this be configurable by the user? They have already configured `getAttributeName()`, but maybe they need specific `objectclass` filters?

Reply:
@dwnusbaum `user` and ìnetOrgPerson` are valid but selective objectclass for configured users. I recommend you to read [this article](https://social.technet.microsoft.com/wiki/contents/articles/20289.inetorgperson-class-in-active-directory.aspx)